notebooks/WML DB2.ipynb

```diff
@@ -47,26 +48,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
-    "#PLEASE ENTER YOU OWN CREDENTIALS TO GET DATA FROM DATABASE. \n",
-    "#CLICK THE 'FILE AND DATA' BUTTON THE TOP RIGHT SIDE OF THE MENU BAR\n",
-    "\n",
-    "\n",
-    "#from ibmdbpy import IdaDataBase, IdaDataFrame\n",
-    "\n",
-    "# @hidden_cell\n",
-    "# This connection object is used to access your data and contains your credentials.\n",
-    "# You might want to remove those credentials before you share your notebook.\n",
-    "#idadb_fe06e3df5eef4ac0b1c246b40ed3db6e = IdaDataBase(dsn='DASHDB;Database=BLUDB;Hostname=dashdb-txn-sbox-yp-dal09-03.services.dal.bluemix.net;Port=50000;PROTOCOL=TCPIP;UID=pcl47325;PWD=mwhpp-qlc4wf54s4')\n",
-    "\n",
-    "#pd_df = IdaDataFrame(idadb_fe06e3df5eef4ac0b1c246b40ed3db6e, 'PCL47325.HOME_SALES').as_dataframe()\n",
-    "#pd_df.head()\n",
-    "\n",
-    "# You can close the database connection with the following code. Please keep the comment line with the @hidden_cell tag,\n",
-    "# because the close function displays parts of the credentials.\n",
-    "# @hidden_cell\n",
-    "# idadb_fe06e3df5eef4ac0b1c246b40ed3db6e.close()\n",
-    "# To learn more about the ibmdby package, please read the documentation: http://pythonhosted.org/ibmdbpy/\n"
+    "# Connect to datasource\n"
```

Comment:
not sure if you meant to delete the "PLEASE ENTER..." comment. Your readme probably has it covered.